### PR TITLE
Add base shipping price to order

### PR DESCRIPTION
**Pull request by fowczarek**
I want to merge this change because of adding the base shipping price to the order.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-7786 Create shipping base price in Order](https://app.clickup.com/t/2549495/SALEOR-7786)

File: saleor/order/migrations/0158_migrate_base_shipping_price_amount.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        override_global_tax = config_dict.get("override_global_tax")
+        if override_global_tax:
+            include_taxes_in_prices = config_dict.get("include_taxes_in_prices")
+            if include_taxes_in_prices:
+                channel_ids_with_tax_included.append(config.channel_id)
+            else:
+                channel_ids_with_tax_excluded.append(config.channel_id)
+
+    # Fetch channel ids with default settings
+    channel_ids_with_overrode_config = (
```
Comment (IKarbowiak) [inline saleor/order/migrations/0158_migrate_base_shipping_price_amount.py:33]: Typo `channel_ids_with_override_config`

File: saleor/order/migrations/0158_migrate_base_shipping_price_amount.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        channel_ids_with_tax_included + channel_ids_with_tax_excluded
+    )
+    site_settings = SiteSettings.objects.first()
+    channel_ids_without_overrode_config = Channel.objects.exclude(
```
Comment (IKarbowiak) [inline saleor/order/migrations/0158_migrate_base_shipping_price_amount.py:37]: Typo `channel_ids_without_override_config`

**Review by IKarbowiak — APPROVED** [review]
🚀

**Review by jakubkuc — APPROVED** [review]

**Pull request merged**